### PR TITLE
FIX for List available backends

### PR DESCRIPTION
**Pull request by plaindocs**
Fix for Issue #1271.

### CODE REVIEW

File: lib/asciidoctor/cli/options.rb [hunk 1]
```diff
@@ -43,7 +43,8 @@ def parse!(args)
 
           EOS
 
-          opts.on('-b', '--backend BACKEND', 'set output format backend (default: html5)') do |backend|
+          opts.on('-b', '--backend BACKEND', 'set output format backend: [html5, xhtml, docbook, docbook45] (default: html5)',
+                  'Other backends such as [manpage, pdf, deckjs] can be added') do |backend|
```
Comment (mojavelinux) [inline lib/asciidoctor/cli/options.rb]: Could we write the second line as follows?

```
additional backends are supported via extensions (e.g., pdf, manpage, latex)
```

Note the lowercase at the start of the sentence. That keeps it consistent with how other options are written.

**Comment by mojavelinux** [comment]
Thanks! I made one small suggestion.

**Comment by mojavelinux** [comment]
We now have a built-in manpage backend coming in 1.5.3. Could you add that to the list (at the end). Then I'll merge.

**Comment by plaindocs** [comment]
Done.

**Comment by mojavelinux** [comment]
Great! Thanks @plaindocs!

**Pull request merged**